ui/app/models/host-volume.js

```diff
@@ -10,6 +10,8 @@ import { fragmentOwner } from 'ember-data-model-fragments/attributes';
 export default class HostVolume extends Fragment {
   @fragmentOwner() node;
 
+  @attr('string') volumeID;
```

Comment:
When we serialize a single node, `node.HostVolumes` come back as an array of this model type: a fragment called Host Volume.

In all other places where we'd show dynamic host volumes (individual dhv page, or on the storage index page), these are appropriately loaded as dynamic host volume models (non-fragment).

It's important to have volumeID declared here, though, so we can filter out the Dynamic host volumes from the static ones when the user does a `scan` on the storage page (since `clients[].each.HostVolumes[]` wouldn't have a property to filter out otherwise)